client/app/components/text-area.js

```diff
@@ -0,0 +1,9 @@
+import Component from '@glimmer/component';
+
+export default class TextAreaComponent extends Component {
```

Comment:
maxLength here has to be an argument because the CharacterCounter component API requires that as an argument.  

Ah, yeah you're right, we can rename maxlength to be without the camelcase.

Reply:
Ah okay, I think I understand. I'm confusing the <textarea> maxlength attr with our own component's maxlength argument.